packages/protoplugin/src/ecmascript/generated-file.ts

```diff
@@ -37,6 +37,27 @@ type Printable =
   | DescEnum
   | Printable[];
 
+export interface TSFile {
```

Comment:
Temporary name because I couldn't think of one better.  I considered `OutFile`, but not sure if that's best.